usr/share/rear/skel/NBKDC/etc/init/rcmd-executor.conf

```diff
@@ -0,0 +1,8 @@
+description "NBK DC Remote Command Executor"
+start on runlevel [2345]
+stop on runlevel [016]
+
+respawn
+normal exit 0
+
+exec /opt/NovaStor/DataCenter/rcmd-executor/rcmd-executor run
```

Comment:
Correct, thanks for pointing that out.
That should not be hard-coded, I guess I took it from a default installaation and just copied it here. 
Perhaps I can use the NBKDC_DIR variable here